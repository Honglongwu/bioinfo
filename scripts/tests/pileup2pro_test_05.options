 --in pileup2pro_test_05.input --has-mapping-quality --which-bam 1,5
