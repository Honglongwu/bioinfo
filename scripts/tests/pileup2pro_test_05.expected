>seq1-s
1	0	0	0	0
2	0	0	0	0
3	0	0	0	0
4	0	0	0	0
5	0	0	0	0
6	0	0	0	0
7	0	0	0	0
8	0	0	0	0
9	0	0	0	0
10	0	0	0	0
11	0	0	0	0
12	0	0	0	0
13	0	0	0	0
14	0	0	0	0
15	0	0	0	0
16	0	0	0	0
17	0	0	0	0
18	0	0	0	1
19	0	0	0	1
20	0	0	1	0
21	0	0	0	1
22	0	1	0	0
23	0	1	0	0
24	0	0	0	1
25	0	0	0	2
26	0	0	2	0
27	0	0	0	2
28	0	0	0	2
29	0	2	0	0
30	0	0	0	2
31	0	0	0	2
32	2	0	0	0
33	0	0	0	2
34	0	0	0	2
35	0	2	0	0
36	0	0	0	2
37	0	0	2	0
38	0	0	2	0
39	0	0	0	2
40	0	0	0	2
41	0	0	0	2
42	2	0	0	0
43	0	0	2	0
44	0	2	0	0
45	0	0	0	2
46	2	0	0	0
47	2	0	0	0
48	2	0	0	0
49	3	0	0	0
50	0	0	0	3
